backend/data/src/main/kotlin/io/tolgee/configuration/tolgee/WebsocketProperties.kt

```diff
@@ -0,0 +1,8 @@
+package io.tolgee.configuration.tolgee
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "tolgee.websocket")
+class WebsocketProperties(
+  var useRedis: Boolean = false,
```

Comment:
Is keeping this configurable really meaningful? If a Redis server is configured, I think there is nothing wrong with automatically enabling Redis routing, and falling back to standard routing when Redis is not configured. I'm not so sure about the way things are handled internally, but I'd assume this has not much impact on the dispatch performance?

Reply:
I would keep it configurable. The Redis configuration depends on this prop as well. 

See: kotlin/io/tolgee/configuration/ConditionalRedissonAutoConfiguration.kt
```
@Configuration
@ConditionalOnClass(Redisson::class, RedisOperations::class)
@AutoConfigureBefore(RedisAutoConfiguration::class)
@EnableConfigurationProperties(RedissonProperties::class, RedisProperties::class)
@ConditionalOnExpression(
  "\${tolgee.websocket.use-redis:false} or " +
    "(\${tolgee.cache.use-redis:false} and \${tolgee.cache.enabled:false})"
)
class ConditionalRedissonAutoConfiguration : RedissonAutoConfiguration()
```

To fix this I would have to create some global prop like "tolgee.redis.enabled" to enable Redis. But maybe user would like to use redis only for caching, since they have only single instance and not for websocket pub/sub.